tests/conftest.py

```diff
@@ -1,14 +1,29 @@
-import gc
-import os
 import pytest
-import pyvista
 from pyvista.plotting import system_supports_plotting
 
 NO_PLOTTING = not system_supports_plotting()
 
+
+def _check_qt_installed():
+    try:
+        from qtpy import QtCore  # noqa
+    except Exception:
+        return False
+    else:
+        return True
+
+
 @pytest.fixture()
 def plotting():
     """Require plotting."""
     if NO_PLOTTING:
         pytest.skip(NO_PLOTTING, reason="Requires system to support plotting")
     yield
+
+
+@pytest.fixture()
+def no_qt(monkeypatch):
+    """Require plotting."""
+    if _check_qt_installed():
+        monkeypatch.setenv('QT_API', 'bad_name')
```

Comment:
Indeed, I think in addition to this you'll need to force reloading of the module somehow if possible, maybe via `importlib.reload`?

Reply:
(you can call it after the monkeypatch, then again after the `yield` and maybe this is enough)

Reply:
Oh nice! I did not know it was possible.

Reply:
But now it fails inside the `no_qt` fixture!

Reply:
That's fine I think, just `try: except: pass` it. Then the `from pyvistaqt import ...` will hit the same exception in the test.